src/component/ViewHeader/index.jsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function ViewHeader({idName}) {
```

Comment:
ViewHeader isn't the best name, maybe something like ViewInvoiceHeader would be better